### PR TITLE
Fix sidebar navigation and Products Page overlapping problem

### PR DESCRIPTION
**Pull request by kamilpastuszka**
This PR is to fix content overlapping issue between sidebar navigation and Products Page.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4412 Sidebar navigation and Products Page overlapping problem](https://app.clickup.com/t/2549495/SALEOR-4412)

File: src/products/components/ProductListPage/ProductListPage.tsx [hunk 1]
```diff
@@ -64,6 +64,11 @@ const useStyles = makeStyles(
     },
     settings: {
       marginRight: theme.spacing(2)
+    },
+    container: {
+      [theme.breakpoints.up("md")]: {
+        marginLeft: "211px"
```
Comment (orzechdev) [inline src/products/components/ProductListPage/ProductListPage.tsx:70]: Use `drawerWidthExpanded` from `@saleor/components/AppLayout/consts`

**Review by orzechdev — CHANGES_REQUESTED** [review]

File: src/products/components/ProductListPage/ProductListPage.tsx (hunk 1 above)
Comment (bmigirl) [reply src/products/components/ProductListPage/ProductListPage.tsx:70]: What he said ☝️

Comment (kamilpastuszka) [reply src/products/components/ProductListPage/ProductListPage.tsx]: done

**Review by orzechdev — APPROVED** [review]

**Comment by bmigirl** [comment]
There's still this one case where the table itself overlaps, I think it could be full page width at this point. @pwgryglak wdyt?
Screenshot 2021-09-15 at 10 23 44

**Comment by pwgryglak** [comment]
> There's still this one case where the table itself overlaps, I think it could be full page width at this point. @pwgryglak wdyt?
> Screenshot 2021-09-15 at 10 23 44

**Comment by kamilpastuszka** [comment]
@bmigirl What breakpoint was it?
Overlapping text in the table is fixed in a different PR.

**Comment by bmigirl** [comment]
@kamilpastuszka I didn't check, so please do

**Comment by kamilpastuszka** [comment]
@bmigirl Could you check again? It seems don’t have that issue.

**Comment by bmigirl** [comment]
@kamilpastuszka still happening in the deployment
Screenshot 2021-09-21 at 15 15 21

**Comment by kamilpastuszka** [comment]
@bmigirl Can you confirm the breakpoint?

**Comment by bmigirl** [comment]
@kamilpastuszka do you need instruction on how to debug this?

**Comment by kamilpastuszka** [comment]
@bmigirl In order to verify it correctly I need to make sure we are looking at the same screen size. This is why I asked you to confirm the breakpoint/screen width.

**Comment by bmigirl** [comment]
@kamilpastuszka `lg`

**Comment by kamilpastuszka** [comment]
This is how it looks at Material UI lg breakpoint. Here 1200px.

products-lg-breakpoint

**Review by jwm0 — APPROVED** [review]

**Review by bmigirl — APPROVED** [review]

**Pull request merged**